src/App.js

```diff
@@ -3,6 +3,9 @@ import 'react-toastify/dist/ReactToastify.css'
 
```

Comment:
Please pull the latest and make the necessary changes to this file

Reply:
I believe that should also fix the conflicts

Reply:
Okay